client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -64,84 +57,30 @@ const DesignSelector: React.FunctionComponent = () => {
-							<button
-								key={ design.slug }
-								className="design-selector__design-option"
-								data-e2e-button={ design.is_premium ? 'paidOption' : 'freeOption' }
-								onClick={ () => {
-									setSelectedDesign( design );
+			<DesignPicker
+				designs={ getRandomizedDesigns().featured.filter(
+					( design ) =>
+						// TODO Add finalized design templates to available-designs-config.json
+						// along with is_anchorfm prop
+						isAnchorFmSignup === design.features.includes( 'anchorfm' )
+				) }
+				isGridMinimal={ isAnchorFmSignup }
+				locale={ locale }
+				onSelect={ ( design: Design ) => {
+					setSelectedDesign( design );
 
-									// Update fonts to the design defaults
-									setFonts( design.fonts );
+					// Update fonts to the design defaults
+					setFonts( design.fonts );
 
-									goNext();
-								} }
-							>
-								<span
-									className={ classnames(
-										'design-selector__image-frame',
-										isEnabled( 'gutenboarding/landscape-preview' )
-											? 'design-selector__landscape'
-											: 'design-selector__portrait',
-										design.preview === 'static'
-											? 'design-selector__static'
-											: 'design-selector__scrollable'
-									) }
-								>
-									{ isEnabled( 'gutenboarding/mshot-preview' ) ? (
-										<MShotsImage
-											url={ getDesignUrl( design, locale ) }
-											aria-labelledby={ makeOptionId( design ) }
-											alt=""
-											options={ mShotOptions() }
-										/>
-									) : (
-										<img
-											alt=""
-											aria-labelledby={ makeOptionId( design ) }
-											src={ getDesignImageUrl( design ) }
-										/>
-									) }
-								</span>
-								<span className="design-selector__option-overlay">
-									<span id={ makeOptionId( design ) } className="design-selector__option-meta">
-										<span className="design-selector__option-name">{ design.title }</span>
-										{ design.is_premium && (
-											<Tooltip
-												position="bottom center"
-												text={ __( 'Requires a Personal plan or above' ) }
-											>
-												<div className="design-selector__premium-container">
-													<Badge className="design-selector__premium-badge">
-														<JetpackLogo
-															className="design-selector__premium-badge-logo"
-															size={ 20 }
-														/>
-														<span className="design-selector__premium-badge-text">
-															{ __( 'Premium' ) }
-														</span>
-													</Badge>
-												</div>
-											</Tooltip>
-										) }
-									</span>
-								</span>
-							</button>
-						) ) }
-				</div>
-			</div>
+					goNext();
+				} }
+				premiumBadge={
+					<Badge className="design-selector__premium-badge">
+						<JetpackLogo className="design-selector__premium-badge-logo" size={ 20 } />
+						<span className="design-selector__premium-badge-text">{ __( 'Premium' ) }</span>
+					</Badge>
+				}
```

Comment:
This badge is not used in production and it can be displayed only by manually switching `is_premium` property to `true` in the `available-designs-config.json` for a design. Here's a screen capture with the badge:
<img width="310" alt="Screenshot 2021-03-24 at 15 02 07" src="https://user-images.githubusercontent.com/14192054/112328710-43426900-8cbf-11eb-9c58-a89c91d3b5cd.png">